index.html

```diff
@@ -716,389 +716,343 @@ <h2>Identifier</h2>
-      </p>
-
-    </section>
-
-    <section>
-      <h2>Path</h2>
-
-      <p>
-A generic <a>DID path</a> is identical to a URI path and MUST conform to the
-<code>path-abempty</code> ABNF rule in [[!RFC3986]]. A <a>DID path</a> SHOULD be
-used to address resources available through a <a>service endpoint</a>. For more
-information, see Section <a href="#service-endpoints"></a>.
-      </p>
-
-      <p>
-A specific <a>DID scheme</a> MAY specify ABNF rules for <a>DID paths</a> that
-are more restrictive than the generic rules in this section.
-      </p>
-
-      <pre class="example nohighlight">
-did:example:123456/path
-      </pre>
-    </section>
-
-    <section>
-      <h2>Query</h2>
-
-      <p>
-A generic <a>DID query</a> is identical to a URI query and MUST conform to the
-<code>query</code> ABNF rule in [[!RFC3986]]. A <a>DID query</a> SHOULD be used
-to address resources available through a <a>service endpoint</a>. For more
-information, see Section <a href="#service-endpoints"></a>.
-      </p>
-
-      <p>
-A specific <a>DID scheme</a> MAY specify ABNF rules for <a>DID queries</a> that
-are more restrictive than the generic rules in this section.
-      </p>
-
-      <pre class="example nohighlight">
-did:example:123456?query=true
-      </pre>
-    </section>
-
-    <section>
-      <h2>Fragment</h2>
-
-      <p>
-A <a>DID fragment</a> is used as method-independent reference into the <a>DID
-document</a> to identify a component of the document (for example, a unique
-<a>public key description</a> or <a>service endpoint</a>). <a>DID fragment</a>
-syntax and semantics are identical to a generic URI fragment and  MUST conform
-to <a data-cite="RFC3986#section-3.5">RFC&nbsp;3986, section 3.5</a>. To resolve
-a <a>DID fragment</a> reference, the complete <a>DID URL</a> including the
-<a>DID fragment</a> MUST be used as input to the <a>DID URL</a> dereferencing
-algorithm for the target component in the <a>DID document</a> object. For more
-information, see [[DID-RESOLUTION]].
-      </p>
-
-      <p>
-A specific <a>DID scheme</a> MAY specify ABNF rules for <a>DID fragments</a>
-that are more restrictive than the generic rules in this section.
-      </p>
-
-      <p>
-Implementations need not rely on graph-based processing of <a>DID documents</a>
-to locate metadata contained in the <a>DID document</a> when the <a>DID</a>
-includes a <a>DID fragment</a>. Tree-based processing can be used instead.
-      </p>
-
-      <p>
-Implementations SHOULD NOT prevent the use of <a>JSON Pointer</a>
-([[!RFC6901]]).
-      </p>
-
-      <pre class="example nohighlight">
-did:example:123456#public-key-1
-      </pre>
-
-      <p>
-Additional semantics for fragment identifiers, which are compatible with and
-layered upon the semantics in this section, are described for JSON-LD
-representations in Section <a href="#application-did-ld-json"></a>.
-      </p>
-
-    </section>
+      <h3>DID Identifiers</h3>
```

Comment:
```suggestion
      <h3>DID Syntax</h3>
```
DID stands for "Decentralized Identifier", so the heading is strange "Decentralized Identifier Identifiers"

Reply:
Done in https://github.com/w3c/did-core/pull/212/commits/9c6b3618db428edc90ebd42e1e28d77059560867